crates/aptos-in-memory-cache/src/caches/fifo.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{Cache, Incrementable, Ordered};
+use dashmap::DashMap;
+use parking_lot::RwLock;
```

Comment:
out of curiosity, why parking_lot over tokio native RwLock?

Reply:
parking_lot for sync, otherwise I think tokio RwLock for async

Reply:
can we add a comment here w/ this justification?